content/applications/productivity/studio/automated_actions.rst

```diff
@@ -0,0 +1,216 @@
+:show-content:
+
+.. _studio/automated-actions:
+
+===============================
+Automated actions (automations)
+===============================
+
+Automated actions are used to trigger automatic changes based on user actions (e.g., apply a
+modification when a field is set to a specific value) or on time conditions (e.g., archive a record
+7 days after its last update).
+
+To create an automated action with Studio, the following elements need to be defined: the
+:ref:`automated-actions/model`, the :ref:`automated-actions/trigger`, the :ref:`"Apply on"
+<automated-actions/apply-on>`, and the :ref:`automated-actions/action`.
+
+.. _automated-actions/model:
+
+Model
+=====
+
+Select the model where the automated action should be applied.
+
+.. _automated-actions/trigger:
+
+Trigger
+=======
+
+Define when the automated action should be applied. Different triggers are available.
+
+.. _trigger/on-creation:
+
+On Creation
+-----------
+
+The action is triggered when a record is created and then saved.
+
+.. _trigger/on-update:
+
+On Update
+---------
+
+The action is triggered when a previously saved record is edited and then saved.
+
+- Use :guilabel:`Trigger Fields` to specify which fields - and only those - trigger the action on
+  their update.
+- To detect when a record changes from one state to another, define a :guilabel:`Before Update
+  Domain` filter, which checks if the condition is satisfied before the record is updated. Then set
+  an :ref:`automated-actions/apply-on` filter, which checks if the condition is satisfied after the
+  record is updated.
+
+  .. example::
+     If you want the automated action to happen when an email address is set on a contact, define
+     the :guilabel:`Before Update Domain` to: ``email is not set`` and the :guilabel:`Apply on`
+     domain to: ``email is set``.
+
+.. _trigger/on-creation-update:
+
+On Creation & Update
+--------------------
+
+The action is triggered when a record is created and saved or edited afterward and saved.
+
+.. tip::
+   Add a Condition, or the automated action might run non-stop.
```

Comment:
What could be the conditions? How to add a condition?

Reply:
I'm removing that tip as I'm not sure how useful it is - but your comment was relevant :)